app/controllers/sulten/lyche_controller.rb

```diff
@@ -24,6 +24,10 @@ def reservation_failure
 
   end
 
+  def reservation_failure_day
```

Comment:
This name didn't make sense to me. Could you spare a comment about it?

Reply:
Added!

Reply:
Thanks! It's little comments like that that gives me context when diving deeper into the PR. :)